test/lib/circuit/test/lib/asserter/README.md

```
@@ -0,0 +1,33 @@
+# asserter
+
+![Verify Assertion Failure](https://github.com/mucbuc/asserter/workflows/Verify%20Assertion%20Failure/badge.svg)
+
+assertion util inspired by John Torjo's solution described [here](https://www.drdobbs.com/cpp/enhancing-assertions/184403745)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)